modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -9,16 +9,25 @@
 
 Adding the {microshift-short} RPM package to an Image Builder blueprint enables the build of a {op-system-ostree} image with {microshift-short} embedded.
 
+* Follow step 1 to create your own minimal blueprint file which results in a faster minimal {microshift-short} installation.
+* Follow Step 2 to use the generated blueprint for installation which includes all the RPM packages and container images. This is a longer installation process, but a faster start up because container references are accessed locally.
++
+[IMPORTANT]
+====
+* Replace _<microshift_blueprint.toml>_ in the following procedures with the name of the TOML file you are using.
+* Replace _<microshift_blueprint>_ in the following procedures with the name you want to use for your blueprint.
+====
+
 .Procedure
 
-. Use the following example to create your blueprint:
+. Use the following example to create your own blueprint file:
 +
-.Image Builder blueprint example
+.Custom Image Builder blueprint example
 +
 [source,text]
 ----
-cat > minimal-microshift.toml <<EOF
-name = "minimal-microshift"
+cat > microshift_blueprint.toml <<EOF <1>
```

Comment:
```suggestion
cat > _<microshift_blueprint.toml>_ <<EOF <1>
```

Reply:
This is an actual image builder blueprint example. so can this have <>?